### ISSUE TITLE
PATCH /sale/product-offers i "Missing required parameters"

### ISSUE THREAD

**Issue by tomylee0** [allegro/allegro-api#9432]
Nie jestem pewien czy dobrze zrozumiałem #9193 
Bo np. tutaj przy PATCH'u /sale/product-offers otrzymuję błąd: 
Missing required parameters: 235009
(trace-id: 721e96872477122b)

A ten fragment z #9193 mówi mi, że i ilość sztuk i cenę można zmieniać nawet przy braku obowiązkowego parametru:

> Oznacza to, że - **podobnie jak w przypadku zmiany liczby sztuk** - nawet jeśli w ofercie będzie brakować wymaganego parametru, to samą operację zmiany ceny przeprowadzimy z pozytywnym rezultatem.

**Comment by MartaNowaczyk** [comment]
Tak, ale dzieje się tak w przypadku zmiany samej liczby sztuk, a nie w przypadku podania jeszcze innych pól. W podanym przykładzie oprócz stocku zmieniasz jeszcze "taxSettings".

**Comment by tomylee0** [comment]
Ok. W tym przypadku tak rzeczywiście było. 

Skoro jednak można zmieniać ilość lub cenę nawet gdy nie ma obowiązkowego parametru to mam wątpliwości co do zabraniania modyfikacji taxSettings. TaxSettings jest mocno powiązany z ceną. Czasem ktoś wpisze złą stawkę VAT'u. Poprawienie stawki VAT wymusza poprawę ceny w ofercie. I tu spotkamy się z blokadą. Jeśli więc można zmieniać ilość i cenę to może nie byłby to problem, żeby również taxSettings można było zmieniać?

Pozdrawiam.

**Comment by MartaNowaczyk** [comment]
W taki sposób można wiele rzeczy/pól połączyć i każdy mógłby chcieć co innego. W każdym razie dwa kluczowe parametry oferty można zmienić, a na tym nam zależało. 
Pamiętaj, że zawsze możesz dodać swoją sugestię do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions). 
Pozdrawiam
